paparazzi-gradle-plugin/src/test/projects/verify-resources-modules/module2/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
i'm confused; shouldn't there be at least one string resource defined in each of these modules?

Reply:
As discussed, we just need dependency of the module and test the order, it does not matter if the module has actual resource or not

Reply:
As discussed, we just need dependency of the module and test the order, it does not matter if the module has actual resource or not

Reply:
As discussed, we just need dependency of the module and test the order, it does not matter if the module has actual resource or not